css/style.css

```diff
@@ -0,0 +1,715 @@
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@200;300;400;500;600;700&display=swap');
+
+body {
+  font-family: 'Oswald', sans-serif;
+  font-style: normal;
+  font-weight: 100;
+  max-width: 1920px;
+  height: 100px;
+  margin: 0;
+  padding: 0;
+  color: #333;
+  overflow-x: hidden;
+}
+
+.mobile-img {
+  display: none;
+}
+
+.menumobile {
+  display: none;
+}
+
+.top-header {
+  display: flex;
+  width: 100%;
+  position: absolute;
+  max-width: 1920px;
+  height: 36px;
+  justify-content: flex-end;
+  align-items: center;
+  box-sizing: border-box;
+  background-color: #272a31;
+}
+
+.header-content {
+  display: flex;
+  position: absolute;
+  width: 45%;
+  align-content: center;
+}
+
+.header-list {
+  list-style-type: none;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.header-list li {
+  justify-content: space-between;
+  margin: 0 20px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 24px;
+  text-align: right;
+  color: #fff;
+}
+
+.navigation {
+  display: flex;
+  width: 100%;
+  position: relative;
+  justify-content: space-evenly;
+  align-items: center;
+  box-sizing: border-box;
+  align-content: center;
+  padding: 10px;
+  box-shadow: 0 10px 18px #888;
+  top: 36px;
+  height: 80px;
+}
+
+.logo,
+.logo-img {
+  vertical-align: middle;
+  display: flex;
+  justify-content: space-between;
+  height: 95%;
+  cursor: pointer;
```

Comment:
Could you explain for me? Because the "real one" is like that: https://summit.cckorea.org/introduce.html
![image](https://user-images.githubusercontent.com/84752859/129265482-ea27d0a9-e894-49c2-8f7e-22e158a24a25.png)

Reply:
@leonardoeng13 please follow these guidelines: https://www.behance.net/gallery/29845175/CC-Global-Summit-2015

that's not the original website, but a modified version of the template in the link above, in the project requirements we ask you to follow the template, not the website.

Reply:
:heavy_check_mark:  Logo doesn't appear on mobile resolution now.